index.html

```diff
@@ -1094,6 +1128,25 @@
 See [[fingerprinting-guidance]] for how to mitigate threats that result from
 [=fingerprinting=].
 
+Supercookies occur when a UA stores data for a site but makes that data more
+difficult to clear than other cookies or storage, typically because of a bug or
+because the browser restores the browser vendor's cookies when local state is cleared.
+<a data-cite="fingerprinting-guidance#clearing-all-local-state">Fingerprinting
+Guidance § Clearing all local state</a> discusses how specifications can help
+UAs avoid this mistake.
+
+<dfn>Header enrichment</dfn> happens when a network operator adds HTTP request headers
+to identify their customers to sites that they visit. It is unfortunately
+difficult for a [=user agent=] to mitigate against [=header enrichment=].
+
+Additionally, [=cross-context recognition=] can happen directly between devices
+in a person's vicinity. Cross-device communication is communication
```

Comment:
I'd keep Nick's phrasing from #186:

```suggestion
<dfn>Cross-device communication</dfn> is communication
```

Reply:
This might be the wrong way to do it but what I'm aiming for here is to tie x-device communication into the broader narrative. Happy to do that some other way but I feel like removing it entirely loses something?

Reply:
Actually I think it works.